build.gradle

```diff
@@ -137,18 +136,15 @@ allprojects {
 
     ext {
         annotationsVersion = "1.1.0"
-        jacksonDatabindVersion = "2.9.9"
-        jacksonDataFormatXmlVersion = "2.9.9"
+        concurrentFuturesVersion = "1.0.0"
         jUnitVersion = "4.12"
-        mockWebServerVersion = "3.13.1"
```

Comment:
Was this removed this so we use the same version as OkHttp?

Reply:
Yes, I condensed a few places where we were using different libraries within the same SDK and had different version variables for each library.